Kill.py

```diff
@@ -0,0 +1,59 @@
+""" Kill Process Extension """
+
+import os
+import getpass
+from albertv0 import *
+from subprocess import Popen, PIPE
+
+__iid__ = "PythonInterface/v0.1"
+__prettyname__ = "Kill Process"
+__version__ = "1.0"
+__trigger__ = "kill"
+__author__ = "Benedict Dudel"
+__dependencies__ = []
+
+def handleQuery(query):
+    if query.isTriggered:
+        results = []
+        for process in getProcesses(query.string.strip()):
+            results.append(
+                Item(
+                    id=process['id'],
+                    icon=iconLookup(process['name']),
+                    text=process['name'],
+                    subtext="CPU: %s %%" % process['cpu'],
+                    completion="",
+                    actions=[
+                        ProcAction("Kill", ["kill", "-SIGKILL", process['id']]),
```

Comment:
I think sigterm is a better default and as alternative action sigkill

Reply:
Sounds good :)